src/editor/editor.directive.ts

```diff
@@ -166,28 +167,49 @@ export class FroalaEditorDirective implements ControlValueAccessor {
     if (!this._opts.events) {
       this._opts.events = {};
     }
+
+    // Store user event callbacks if exist
+    if (this._opts.events[eventName]) {
+      this._userEventCallbacks[eventName] = this._opts.events[eventName];
+    }
+
     this._opts.events[eventName] = callback;
   }
 
   private initListeners() {
     let self = this;
 
     // bind contentChange and keyup event to froalaModel
-    this.registerEvent(this._element, "froalaEditor.contentChanged", function() {
+    this.registerEvent(this._element, 'contentChanged', function() {
       setTimeout(function() {
         self.updateModel();
+        if (self._userEventCallbacks['contentChanged'] && self._editor.events) {
+          self._editor.events.on('contentChanged', self._userEventCallbacks['contentChanged']);
+          self._userEventCallbacks['contentChanged']();
+          self._userEventCallbacks['contentChanged'] = null;
+        }
       }, 0);
     });
-    this.registerEvent(this._element, "froalaEditor.mousedown", function() {
+    this.registerEvent(this._element, 'mousedown', function() {
       setTimeout(function() {
         self.onTouched();
+        if (self._userEventCallbacks['mousedown'] && self._editor.events) {
+          self._editor.events.on('mousedown', self._userEventCallbacks['mousedown']);
+          self._userEventCallbacks['mousedown']();
```

Comment:
one we're binding and calling also same time any reason behind it?

Reply:
@navneetnagpal Because we are binding this the very first time the event gets triggered so we need to call the function as well for this time as this won't be triggered the first time.